<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkRadiomicsArithmetricViewControls</class>
 <widget class="QWidget" name="QmitkRadiomicsArithmetricViewControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>980</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="m_InputImageGroup"/>
   </item>
   <item row="5" column="0">
    <widget class="QToolBox" name="toolBox">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>430</width>
        <height>852</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Single Image Operations</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QCommandLinkButton" name="buttonSin">
           <property name="text">
            <string>Sinus</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCommandLinkButton" name="buttonASin">
           <property name="text">
            <string>A-Sinus</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCommandLinkButton" name="buttonCos">
           <property name="text">
            <string>Cosinus</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCommandLinkButton" name="buttonACos">
           <property name="text">
            <string>A-Cosinus</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCommandLinkButton" name="buttonTan">
           <property name="text">
            <string>Tangents</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCommandLinkButton" name="buttonATan">
           <property name="text">
            <string>A-Tangents</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCommandLinkButton" name="buttonSquare">
           <property name="text">
            <string>Square</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCommandLinkButton" name="buttonSqrt">
           <property name="text">
            <string>Square-Root</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCommandLinkButton" name="buttonAbs">
           <property name="text">
            <string>Absolute Value</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCommandLinkButton" name="buttonExp">
           <property name="text">
            <string>Exponential (exp)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCommandLinkButton" name="buttonExpNeg">
           <property name="text">
            <string>Negative Exponential</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCommandLinkButton" name="buttonLog10">
           <property name="text">
            <string>Logarithmic (Base 10)</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>430</width>
        <height>852</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Single Image and Double Value Operation</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QFormLayout" name="formLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Additional Value: </string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QDoubleSpinBox" name="m_DoubleValue">
             <property name="decimals">
              <number>4</number>
             </property>
             <property name="minimum">
              <double>-1000000000000000000000.000000000000000</double>
             </property>
             <property name="maximum">
              <double>1000000000000000000000.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCommandLinkButton" name="buttonAddLeft">
         <property name="text">
          <string>Add (Image + Value)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCommandLinkButton" name="buttonSubLeft">
         <property name="text">
          <string>Subtract (Image - Value)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCommandLinkButton" name="buttonSubRight">
         <property name="text">
          <string>Subtract (Value - Image)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCommandLinkButton" name="buttonMulLeft">
         <property name="text">
          <string>Multiply (Image x Value)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCommandLinkButton" name="buttonDivLeft">
         <property name="text">
          <string>Divide (Image / Value)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCommandLinkButton" name="buttonDivRight">
         <property name="text">
          <string>Divide (Value / Image)</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>430</width>
        <height>852</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Two Image Operations</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QFormLayout" name="formLayout_2"/>
         </item>
         <item>
          <widget class="QWidget" name="secondImageWidget" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>10</height>
            </size>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_9"/>
          </widget>
         </item>
         <item>
          <widget class="QCommandLinkButton" name="buttonAdd">
           <property name="text">
            <string>Add (Image + Second Image)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCommandLinkButton" name="buttonSub">
           <property name="text">
            <string>Subtract (Image - Second Image)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCommandLinkButton" name="buttonMul">
           <property name="text">
            <string>Multiply (Image x Second Image)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCommandLinkButton" name="buttonDiv">
           <property name="text">
            <string>Divide (Image - Second Image)</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="1">
      <widget class="QCheckBox" name="m_ResultAsDouble">
       <property name="text">
        <string/>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Return result as double: </string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
